.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+name: Release
+# only trigger on tags, `verify` has already been triggered by push to PR
+on:
+  push:
+    tags: ["v[0-9]+.[0-9]+.[0-9]+"]
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
maybe add a name to the checkout action - "Checkout code", but that is small nit-picky detail